queue_processors/autoreduction_processor/autoreduction_processor.py

```diff
@@ -79,7 +79,7 @@ def hold_message(self, destination, data, headers):
         python_path = sys.executable
         logger.info("Calling: %s %s %s %s",
                     python_path, MISC['post_process_directory'], destination,
-                    message.serialize())    # TODO: limit reduction script  #pylint:disable=fixme
```

Comment:
😆 Seems like writing the todo was the same amount of work than to just do it?
I'm assuming the todo was written before the parameter was added or something? Just looks funny as a PR here, haha

Reply:
Exactly what I thought!